lib/graphql/schema/loader.rb

```diff
@@ -102,7 +102,7 @@ def define_type(type, type_resolver)
               name: type["name"],
               type: type_resolver.call(type["type"]),
               description: type["description"],
-              default_value: type["defaultValue"]
+              default_value: type["defaultValue"] ? JSON.parse(type["defaultValue"], quirks_mode: true) : nil
```

Comment:
I'm pretty sure schema loading only ever worked if you either didn't have `defaultValue`'s or they were actually a string. So it didn't really work :) mine blew up right away due to our usage of ints and enum values as default values.